src/scripts/sssom_to_robot_template.py

```diff
@@ -0,0 +1,57 @@
+"""Convert SSSOM to ROBOT template."""
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Dict, Union
+
+import pandas as pd
+from sssom import MappingSetDataFrame
+from sssom.parsers import parse_sssom_table
+
+ROBOT_ROW = {
+    'subject_id': 'ID',
+    'subject_label': '',
+    'object_id': 'A oboInOwl:hasDbXref',
+    'equivalence': '>A oboInOwl:source',
+    'author_id': '>AI oboInOwl:source SPLIT=|',
+    'mapping_provider': '>A oboInOwl:source',
+    'object_label': '',
+}
+PRED_MAP = {
+    'skos:relatedMatch': 'MONDO:relatedTo',
+    'skos:exactMatch': 'MONDO:equivalentTo',
+    'skos:broadMatch': 'MONDO:mondoIsNarrowerThanSource',
+}
+
+
+def sssom_to_robot_template(inpath: Union[str, Path], outpath: Union[str, Path]):
+    """Convert SSSOM to ROBOT template."""
+    msdf: MappingSetDataFrame = parse_sssom_table(inpath)
+    df: pd.DataFrame = msdf.df
+
+    # Conversion
+    df['equivalence'] = df['predicate_id'].map(lambda pred: PRED_MAP.get(pred, ''))
+    df = df[['subject_id', 'subject_label', 'object_id', 'equivalence', 'object_label']]\
+        .sort_values(['subject_id', 'object_id'])
+    df['author_id'] = '|'.join(msdf.metadata['creator_id'])
+    df['mapping_provider'] = msdf.metadata['mapping_provider']
+    df = pd.concat([pd.DataFrame([ROBOT_ROW]), df])
+
+    # Write
+    df.to_csv(outpath, sep='\t', index=False)
+
+
+def cli():
```

Comment:
- [ ] @matentzn undraft and build or keep drafted

@matentzn The only open thread I see is the [possible `mondolib` refactor](https://github.com/monarch-initiative/mondo-ingest/pull/521#discussion_r1604500855). Other than that, I don't have any todo's left in this PR. If you don't either, and if we opt to do the `mondolib` thing later, then we can undraft this PR and run the build, I think.

Reply:
Dont worry I will take it from here. There are still some issues to iron out which do not relate to you (related to the source mapping fidelity)